src/families/base.js

```diff
@@ -11,6 +11,7 @@ export default class BaseTask extends Component {
 
     uniqueClassName = "base-task";
     call;
+    loadErrors;
```

Comment:
Might be from an old fix of cloning bugs? can you confirm if we still need it or not?